backends/fs/amigaos4/amigaos4-fs.cpp

```diff
@@ -62,13 +62,21 @@ AmigaOSFilesystemNode::AmigaOSFilesystemNode() {
 	_bIsDirectory = true;
 	_sPath = "";
 	_pFileLock = 0;
-	_nProt = 0; // Protection is ignored for the root volume
+	_nProt = 0; // Protection is ignored for the root volume.
 	LEAVE();
 }
 
 AmigaOSFilesystemNode::AmigaOSFilesystemNode(const Common::String &p) {
 	ENTER();
 
+	// We need to explicitely open dos.library and it's IDOS interface.
+	// Otherwise we will hit an IDOS NULL pointer after compiling a shared
+	// binary with (shared) plugins.
+	// The hit will happen on loading a game from any engine, if more
+	// than one engine/plugin is available.
+	DOSBase=IExec->OpenLibrary("dos.library",0);
```

Comment:
Spacing

Reply:
I´m not really sure what to do about the comments...

Reply:
Apparently md5 is referring to the lack of spaces around ‘=‘

Reply:
Ah ok, that should be fixed now.
I was puzzled by the three lines (comments) above (lines 74-76).
Didn't know how i had to fix the spacing there.